docs/upgrade/UPGRADE-unreleased.md

```diff
@@ -154,6 +154,7 @@ for instance:
     - if you have extended `CountryGridFactory`, revise your changes because class changed its namespace
     - if you have extended `CountryFormType`, revise your changes – new fields are available
     - if you have extended `CountryController` revise your changes – `new` and `edit` actions were added
+- if you override method `Localization::getAdminLocale()` to have administration in a different language than english, you can delete it and set parameter `shopsys.admin_locale` in your `parameters.yml` file instead ([#806](https://github.com/shopsys/shopsys/pull/806))
```

Comment:
Without doing this change, the project would still work as intended, so I'd use the *(optional)* tag.

If would have to required if you'd remove the default value of the second argument in the constructor. (As the extending service with overridden method would not inherit the `arguments` configuration from framework's `services.yml`.)

Reply:
According to #803, I didn't want to introduce new optional instruction and assume, that "if you ... you can" phrasing is sufficient.